pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -110,6 +110,18 @@ type BlueGreenStrategy struct {
 	// removing the pause condition.
 	// +optional
 	AutoPromotionSeconds *int32 `json:"autoPromotionSeconds,omitempty"`
+	// MaxUnavailable The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of total pods at the start of update (ex: 10%).
+	// Absolute number is calculated from percentage by rounding down.
+	// This can not be 0 if MaxSurge is 0.
+	// By default, a fixed value of 1 is used.
```

Comment:
We need to figure out what the real default is (what kubernetes chooses). I originally thought it was 20%, then yesterday I thought it was 0, and now I see this comment (which you copied from canary) stating it's 1. But now I'm reading documentation says it's 25%: 
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
The answer is:
* extensions/Deployment defaults it to 1
* apps/Deployment defaults it to 25%
* argoproj.io/Rollout defaults to 0, which is a bug and should be 25%. https://github.com/argoproj/argo-rollouts/issues/938 is tracking that bug.